tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -0,0 +1,33 @@
+# Jobs to run on master after every successful merge is committed
+jobs:
+- job: Windows_Debug_CPU_x64
+  pool: 'Win-CPU'
+  variables:
+    buildDirectory: '$(Build.BinariesDirectory)'
+    buildConfig: 'Debug'
+    buildArch: 'x64'
+
+  steps:
+    - template: templates/set-test-data-variables-step.yml
+    - template: templates/set-version-number-variables-step.yml
+
+    - template: templates/windows-build-tools-setup-steps.yml
+      parameters:
+        EnvSetupScript: 'setup_env.bat'
+        buildArch: 'amd64'  # amd64 is needed for vcvars target arch 
+        setVcvars: false
+
+    - template: templates/windows-build-and-test-steps.yml
+      parameters:
+        buildAdditionalParams: ' --use_openmp '
+        buildArch: $(buildArch)
```

Comment:
Is there a reason we want to use OpenMP here instead of our default internal MLAS implementation?

Reply:
Many of our code can only be parallelled by openmp. Disabling openmp will drop the performance. I have tried

Reply:
Would be good to capture this in a comment in the yaml file

Reply:
comment added